### PR TITLE
spinbox function add in window covering

### PR DESCRIPTION
**Pull request by gdinolee**
Modified to operate when the arrow button of the spinbox is pressed in window covering
Spinbox code clean up

### CODE REVIEW

File: src/things/windowcovering.py [hunk 1]
```diff
@@ -191,27 +200,33 @@ def init_slider(self):
...
     ## Set the window cover to target value ##
-    def to_target(self):
+    def to_target(self, prev_targetlevel = None):
         self.horizontalSliderWindow.setValue(self.targetlevel)
         self.spinBoxWindow.setValue(self.targetlevel)
         if self.targetlevel != self.currentlevel:
             self.send_target_value()
-            self.set_direction()
-            self.timer.start(50)
+            if prev_targetlevel and abs(self.targetlevel - prev_targetlevel) > 1:
```
Comment (junyong-sim) [inline src/things/windowcovering.py:210]: Could you explain why you start timer only when `prev_targetlevel` exists ?

Comment (gdinolee) [reply src/things/windowcovering.py:210]: When adjusting the value with the spin box, the timer does not operate.

File: src/things/windowcovering.py [hunk 2]
```diff
@@ -172,6 +173,14 @@ def set_direction(self):
         elif self.targetlevel < self.currentlevel:
             self.direction = WC_MOVE_DOWN
 
+
```
Comment (jinwon0325kim) [inline src/things/windowcovering.py:176]: remove line

Comment (gdinolee) [reply src/things/windowcovering.py:176]: The change is intended behavior for spacing the above and below functions.

**Review by junyong-sim — APPROVED** [review]
Thanks for hard work

**Pull request merged**